pkg/validate/security_context_linux.go

```diff
@@ -95,8 +95,14 @@ var _ = framework.KubeDescribe("Security Context", func() {
 
 			By("get nginx container pid")
 			command := []string{"cat", "/var/run/nginx.pid"}
-			output := execSyncContainer(rc, containerID, command)
```

Comment:
Sometimes `cat` is executed before `/var/run/nginx.pid` is created by nginx process. So we get no error from execution itself but `cat` reports "not found". So, to be more accurate, `Eventually` here is for automatic retrying rather than waiting.

Reply:
Ah I see now.. thx :-)
maybe try:
`sh -c "while [ ! -f /var/run/nginx.pid ]; do sleep 1; done" && cat /var/run/nginx.pid`

Reply:
Thanks for the comment. I fixed it.